### ISSUE TITLE
Ionic Integration

### ISSUE THREAD

**Issue by denzelwamburu** [ricardodovalle/meanjs-token-auth#1]
Nice stuff :smile: 
Any ideas how to integrate this with ionic app?

**Comment by umairsarfraz** [comment]
@denzelwamburu Any luck with integration in Ionic?

**Comment by denzelwamburu** [comment]
@umairsarfraz  Not Really, i switched to using Loopback, and i integrated it with ionic App. You can check it out here [https://github.com/denzelwamburu/lb-ngIonic] (https://github.com/denzelwamburu/lb-ngIonic)
